### ISSUE TITLE
Import page button

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#633]
The icon for import pages in the editor doesn't look quite right as the other icons are font-awesome (merge looks a bit fuzzy next to them)

Is it worth changing it to a font awesome one too? Not sure which would be most appropriate but here's an example using the download font awesome icon instead (current icon is shown above)...

![icon](https://cloud.githubusercontent.com/assets/8926470/26451419/e7c25dac-4153-11e7-88b0-d69d9a047877.png)

Also, is it necessary for it to be a button here and an option on the insert menu?

**Comment by ronm123** [comment]
@FayCross I think the FA upload icon would be better then the download icon? I guess the alternative would be to swap the current merge icon with an svg version instead. Sebaastian did share an svg version with @torinfo and although it's larger than needed would be easy to resize and would remain as sharp as the FA icons.
[mergeIcon.zip](https://github.com/thexerteproject/xerteonlinetoolkits/files/1029160/mergeIcon.zip)

**Comment by JohnSmith-LT** [comment]
Just to play devils advocate - I don't really like upload or download... I see what the guys were trying to imply; we are really sideloading the pages so a left or right arrow would perhaps be more appropriate.

Can't we make ou own FA icon by overlapping the fa-copy icon with one of the fa left or right arrows?

**Comment by torinfo** [comment]
If we want to move to fa, then I would propose a blank page stacked with the merge icon.

**Comment by JohnSmith-LT** [comment]
Yeah I think the merge icon makes most sense in the short term if we can get the sizing and focus right to make it look like the fa one. I see that the merge icon is in the pipeline to becoming a fa icon so when that happens we can move to using two stacked fa icons but I see little point overlaying a fa with a svg when the svg on its own will suffice...

**Comment by torinfo** [comment]
I've left the merge button as is for now. It is an svg at the moment.
